Allow optional variable to use inventory_hostname instead of ansible_hostname (#38)
sleighzy/ansible-zookeeper: README.md

| zookeeper_election_port                  | 3888                                                              ||
| zookeeper_servers                        | zookeeper-nodes                                                   | See below                                                      |
| zookeeper_use_servers_inventory_hostname | false                                                             | See below                                                      |
| zookeeper_environment                    | "JVMFLAGS": "-javaagent:/opt/jolokia/jolokia-jvm-1.6.0-agent.jar" ||
| zookeeper_firewalld                      | false                                                             ||

## Inventory and zookeeper_servers variable

zookeeper_servers variable above accepts a list of inventory host names.
These will be used in the `zoo.cfg` to configure a multi-server cluster
so the hosts can find each other. By default, the hostname used in
the `zoo.cfg` will be the hostname reported by the `hostname` command on
the server(provided by the ansible_nodename variable). See the example below.

> Assuming the following inventory file and assuming the hostname command for

Reviewer: Can you please rephrase this slightly:

```suggestion
Assuming the below inventory file, and that the `hostname` command returns only the hostname and does not include the domain name.
```

each only gives the hostname without the domain part

```ini
[zookeeper-nodes]
zoo1.foo.com zookeeper_id=1       #hostname command returns "zoo1"
zoo2.foo.com zookeeper_id=2       #hostname command returns "zoo2"
zoo3.foo.com zookeeper_id=3       #hostname command returns "zoo3"
```

And assuming the following role variables:

```yaml
...

>     - role: edgefotwaresolutions.zookeeper

Reviewer: Change the role name:

```suggestion
    - role: sleighzy.zookeeper
```

Author: Applied

      zookeeper_servers:
        - zoo1.foo.com
        - zoo2.foo.com
        - zoo3.foo.com
```

> The `zoo.cfg` will template the following(Assuming all other defaults)

Reviewer: Can you please rephrase this slightly:

```suggestion
The templated `zoo.cfg` file will contain the below entries:
```
